recipes/qt/5.x.x/conanfile.py

```diff
@@ -662,6 +666,43 @@ def package(self):
         for fl in glob.glob(os.path.join(self.package_folder, "lib", "*Qt5Bootstrap*")):
             os.remove(fl)
 
+        for m in os.listdir(os.path.join(self.package_folder, "lib", "cmake")):
+            module = os.path.join(self.package_folder, "lib", "cmake", m, "%sMacros.cmake" % m)
+            if not os.path.isfile(module):
+                tools.rmdir(os.path.join(self.package_folder, "lib", "cmake", m))
+
+
+        extension = ""
+        if self.settings.os == "Windows":
+            extension = ".exe"
+        filecontents = "set(QT_CMAKE_EXPORT_NAMESPACE Qt5)\n"
+        filecontents += "set(QT_VERSION_MAJOR %d)\n" % int(self.version.split(".")[0])
+        filecontents += "set(QT_VERSION_MINOR %d)\n" % int(self.version.split(".")[1])
+        filecontents += "set(QT_VERSION_PATCH %d)\n" % int(self.version.split(".")[2])
+        targets = {}
+        targets["Core"] = ["moc", "rcc", "qmake"]
+        targets["DBus"] = ["qdbuscpp2xml", "qdbusxml2cpp"]
+        if self.options.widgets:
+            targets["Widgets"] = ["uic"]
+        if self.options.qttools:
+            targets["Tools"] = ["qhelpgenerator", "qcollectiongenerator", "qdoc", "qtattributionsscanner"]
+            targets[""] = ["lconvert", "lrelease", "lupdate"]
+        if self.options.qtremoteobjects:
+            targets["RemoteObjects"] = ["repc"]
+        if self.options.qtscxml:
+            targets["Scxml"] = ["qscxmlc"]
+        for namespace, targets in targets.items():
+            for target in targets:
+                filecontents += textwrap.dedent("""\
+                    if(NOT TARGET ${{QT_CMAKE_EXPORT_NAMESPACE}}::{0})
+                        add_executable(${{QT_CMAKE_EXPORT_NAMESPACE}}::{0} IMPORTED)
+                        set_target_properties(${{QT_CMAKE_EXPORT_NAMESPACE}}::{0} PROPERTIES IMPORTED_LOCATION ${{CMAKE_CURRENT_LIST_DIR}}/../../../bin/{0}{1})
+                        set(Qt5{2}_{3}_EXECUTABLE ${{QT_CMAKE_EXPORT_NAMESPACE}}::{0})
```

Comment:
I don't know how it is made in official module file but I think that `Qt5{2}_{3}_EXECUTABLE` should be set to IMPORTED_LOCATION value, it's not exactly the same thing for CMake, specially to interpret this value in add_custom_command

Reply:
I reproduced what's done in qt files: https://github.com/qt/qtbase/blob/v5.15.2/src/dbus/Qt5DBusConfigExtras.cmake.in#L32

Reply:
Great